game_board.py

```diff
@@ -0,0 +1,99 @@
+import pyglet
+
+
+class Cell:
+    def __init__(self, position_x, position_y, cell_size, state):
```

Comment:
Not necessary, but it's a good idea to add a docstring or a comment with expected data types and 'units' - something like:

```
"""
position_x - int - pixel
position_y - int - pixel
cell_size - int - pixel
state - (No idea what should be put here, what are the possible values and how it will be used and it's not evident in the code, so a comment is even more of a good idea).
```

Reply:
Added